server/models/instance-action.js

```diff
@@ -6,6 +6,17 @@ var path = require('path');
 var util = require('util');
 
 module.exports = function extendInstanceAction(InstanceAction) {
+  InstanceAction.observe('after save', function(ctx, next) {
+    // There is no reason to keep these instances around. Until we transition
+    // away from them entirely we'll just auto-expire them.
+    if (ctx.instance && ctx.isNewInstance) {
+      var t = setTimeout(ctx.instance.destroy.bind(ctx.instance), 10 * 1000);
+      if (t.unref) {
```

Comment:
I thought there was a time when they didn't.. but I suppose if there was it was pre-0.10, so I don't need this anyway.